task/02-numbers-tasks.js

```diff
@@ -38,7 +36,7 @@ function getRectangleArea(width, height) {
  *   0    => 0
  */
 function getCicleCircumference(radius) {
-    throw new Error('Not implemented');
+  return radius * 2 * Math.PI;
```

Comment:
JFYI: calculate value of expressions, save it to variable and return variable. Intel/Google styleguide.